### DISCUSSION TITLE
MPEG audio has been replaced by a ragged white noise when exported

### DISCUSSION THREAD

**Discussion by andrewfawside** [mifi/lossless-cut#1715] [Q&A]
MPEG audio has been replaced by a ragged white noise when exported. Any ideas what caused this?

**Comment by mifi** [comment]
does it happen when cutting or regardless? is there any video track or just audio? which file format? tried a different format like matroska?

**Comment by andrewfawside** [comment]
I trimmed either end of an MPEG2 file using version from Microsoft Store. No
format changes or anything like that, just a simple trim

 

Best wishes

Andrew

 

 ***@***.***> ***@***.*** 07785 734992

**Comment by mifi** [comment]
It's hard to say for sure without any sample file, but it's probably a bug in ffmpeg then (which is used by losslesscut). as a workaround you could try to extract each track (video/audio) as separate files, then cut the two files separately, then join the two tracks again.

**Comment by andrewfawside** [comment]
Hi Mikael

I will have a go at that.

 

Best wishes

Andrew

 

 ***@***.***> ***@***.*** 07785 734992
